.eslintrc

```diff
@@ -1,26 +1,32 @@
 {
-  "parser": "babel-eslint",
-  "extends": [
-    "standard",
-    "standard-react"
-  ],
-  "env": {
-    "es6": true
-  },
-  "plugins": [
-    "react"
-  ],
-  "parserOptions": {
-    "sourceType": "module"
-  },
-  "rules": {
-    // don't force es6 functions to include space before paren
-    "space-before-function-paren": 0,
-
-    // allow specifying true explicitly for boolean props
-    "react/jsx-boolean-value": 0,
-
-    // allow comma-dangle in multiline objects & arrays
-    "comma-dangle": ["error", "only-multiline"]
```

Comment:
This line needs to be there in add it back.

    "comma-dangle": ["error", "only-multiline"]